### DISCUSSION TITLE
Error adding custom `gatsbyRemarkPlugins` to `gatsby-plugin-mdx`

### DISCUSSION THREAD

**Discussion by half0wl** [LekoArts/gatsby-themes#1143] [Questions]
I'm trying to add [`gatsby-remark-embed-video`](https://www.gatsbyjs.com/plugins/gatsby-remark-embed-video/) to my Gatsby site. I followed the instructions in https://github.com/LekoArts/gatsby-themes/issues/456#issuecomment-658578530:

* Disable `mdx` (`mdx: false` in theme options)
* Copy-paste the config from https://github.com/LekoArts/gatsby-themes/blob/main/themes/gatsby-theme-minimal-blog-core/gatsby-config.mjs#L31-L51 into my root `gatsby-config.ts`

After running `gatsby clean && gatsby develop`, I ran into a cryptic error that I'm not sure where to start debugging:

```
failed extract queries from components - 0.711s
 ERROR  UNKNOWN

Module not found: Error: Can't resolve
'../../../../../public/page-data/sq/d/3090400250.json'
in '/redacted/node_modules/@lekoarts/g
atsby-theme-minimal-blog/src/hooks'


  ModuleNotFoundError: Module not found: Error: Can't r
  esolve '../../../../../public/page-data/sq/d/30904002
  50.json' in '/redacted/node_modules/
  @lekoarts/gatsby-theme-minimal-blog/src/hooks'

```

Any idea what is going on?

**Comment by melloware** [comment]
Same error here.
